src/cluster/placement/types.go

```diff
@@ -389,17 +387,21 @@ type Options interface {
 	// SetInstrumentOptions sets the instrument options.
 	SetInstrumentOptions(iopts instrument.Options) Options
 
-	// ValidZone returns the zone that added instances must be in in order
-	// to be added to a placement.
+	// Deprecated: Use ZoneOptions (see equivalent method there)
```

Comment:
hm ZoneOptions looks like a cosmetic change to me. i.e. it's pushing down two knobs from Options -> ZoneOptions. Is there another reason to do this? 

if there is, i'd prefer you eliminate the redundancy at the least. i.e. Remove ValidZone/AllowAll.. from Options entirely.

Reply:
Ah yeah fair enough, was mostly thinking of providing a way to allow people to avoid some redundant calls, but agreed that it's not really worth it. Will revert back!

Reply:
Totally fair! Your comment got me thinking a bit more--see the discussion below. tl;dr: I switched this to be SelectorOptions, containing common options for instance selectors.